cafebabel/__init__.py

```diff
@@ -190,5 +190,6 @@ def add_template_helpers():
             lang_url_for=helpers.lang_url_for,
             absolute=helpers.absolute,
             static_pages_for=helpers.static_pages_for,
-            social_network_url_for=helpers.social_network_url_for
+            social_network_url_for=helpers.social_network_url_for,
+            google_analytics_id=app.config.get('GOOGLE_ANALYTICS_ID'),
```

Comment:
You don't need that, `config` is available globally so `config.GOOGLE_ANALYTICS_ID` will work in templates.